[IMP] Expenses: making targeted edits for v 15

larm-odoo opened the pull request:
Needed to edit the pushed new Expenses doc that was sent to v 15  and make edits for the V15 document ONLY.

samueljlieber commented:
Implemented [my technical changes](https://github.com/odoo/documentation/pull/4701#pullrequestreview-1478070068) in https://github.com/odoo/documentation/commit/30b6209c1f4fd978666269e0584a4685f20c7af1. @StraubCreative this PR is ready for your final review, and this PR **should not be FWP to 16.0**.

StraubCreative commented on content/applications/finance/expenses.rst:
```diff
@@ -263,11 +302,15 @@ add, then click :guilabel:`Select`. The items now appear on the report that was
-   :guilabel:`Add a line` only appears when the document is in edit mode. It does not appear
-   otherwise.
+When all edits have been completed, click :guilabel:`Save`. Once the expense report is saved, an
+:guilabel:`Expenes` smart button appears in the top right, listing the number of expenses on the
```
```suggestion
:guilabel:`Expenes` smart button appears in the top-right corner, listing the number of expenses on the
```

StraubCreative commented on content/applications/finance/expenses.rst:
```diff
@@ -365,20 +409,27 @@ in the :guilabel:`Reason to refuse Expense` field, then click :guilabel:`Refuse`
-This presents all the reports for the manager's team.
+Team managers can easily view all the expense reports to approve for their team members. While in
+the :guilabel:`Reports to Approve` view, click on :guilabel:`Filters`, then click :guilabel:`My
+Team`. This presents all the reports to approve for the manager's team. Manager's can easily view
+*all* reports for their team. Simply click the X to remove the :guilabel:`To Approve` filter,
+leaving only the :guilabel:`My Team` filter active.
```
suggestion to join the sentences and remove the **X**
```suggestion
Team`. This presents all the reports to approve for the manager's team. Manager's can easily view
*all* reports for their team by simply clicking the removing the :guilabel:`To Approve` filter in
the :guilabel:`Search...` bar,leaving only the :guilabel:`My Team` filter active.
```

StraubCreative commented on content/applications/finance/expenses.rst:
```diff
@@ -365,20 +409,27 @@ in the :guilabel:`Reason to refuse Expense` field, then click :guilabel:`Refuse`
    If more information is needed, such as a receipt is missing, communication is easy from the
-   chatter. In an individual report, simply type in a message, tagging the proper person (if
-   needed), and post it to the chatter by clicking :guilabel:`Send`. The message is posted in the
-   chatter, and the person tagged will be notified via email of the message, as well as anyone
-   following.
+   chatter. In an individual report, simply click :guilabel:`Send Message`, then type in a message
+   in the text box that appears, tagging the proper person (if needed), and post it to the chatter
+   by clicking :guilabel:`Send`. The message is posted in the chatter, and the person tagged will be
+   notified via email of the message, as well as anyone following.
+
+   Note that only followers can be tagged in a message. To tag someone who is not following, add
+   them as a follower first. Click the :guilabel:`👤 (person)` icon, then click :guilabel:`Add
+   Followers`, then enter their name in the :guilabel:`Recipients` field, and finally, click
+   :guilabel:`Add Followers`.
```
This whole section reads more as a **tip**, actually.
Also would start it like this:
`If more information is needed, such as a missing receipt, communication is  easy from the`

StraubCreative commented on content/applications/finance/expenses.rst:
```diff
@@ -452,9 +502,11 @@ Re-invoice expenses to customers
-customer. This is done by creating an expense report, then creating a sales order with the expensed
-items on it. Then, managers approve the expense report, and the accounting department posts the
-journal entries. Finally, the customer is invoiced.
+customer. This is done by creating an expense, referencing the :abbr:`SO (Sales Order)` the expense
+should be added to, and then create the expense report. Next, managers approve the expense report,
+and the accounting department posts the journal entries. Finally, once the expense report is posted
+to a journal, the expense appears on the :abbr:`SO (Sales Order)` that was referenced. The sales
+order can then be invoiced, thus invoicing the customer for the expense.
```
Reads clunky. Can you give this section another look?

StraubCreative commented on content/applications/finance/expenses.rst:
```diff
@@ -477,37 +529,23 @@ and :guilabel:`Re-Invoicing Policy` by clicking the radio button next to the des
-drop-down menu. Next, select the :guilabel:`Analytic Account` the expense will be posted to.
+entered in order to re-invoice a customer. Select the sales order the expense will appear on in the
+:guilabel:`Customer to Reinvoice` section, from the drop-down menu. Next, select the
+:guilabel:`Analytic Account` the expense will be posted to. After the expense(s) are created, the
```
We introduce the instruction using expense (singular), so grammar should match accordingly.
```suggestion
:guilabel:`Analytic Account` the expense will be posted to. After the expense(s) is created, the
```

StraubCreative commented on content/applications/finance/expenses.rst:
```diff
@@ -519,18 +557,32 @@ set on every expense line of a report. If an :guilabel:`Analytic Account` is mis
 The accounting department is typically responsible for :ref:`posting journal entries
-<expenses/post>`. Once an expense report is approved, it can then be posted.
+<expenses/post>`. Once an expense report is approved, it can then be posted. The :abbr:`SO (Sales
+Order)` is **only** updated *after the journal entries are posted*, and the expenses now appear on
```
mistmatch between singular and plural in this section
```suggestion
Order)` is **only** updated *after the journal entries are posted*, and the expense(s) now appear on
```

StraubCreative reviewed (changes requested):
Hi @larm-odoo 
Looks pretty good here too.
I have a couple of suggestions below.
Overall, double-check singular vs. plural t/o the doc please. I pointed out a couple spots where there was a mismatch, but not all.

larm-odoo commented:
All ready for you, @StraubCreative. I changed up a few sections for the singular/plural issue (the "posting journal entries" was a sneaky one, I caught those!)  I also did some general clean up here and there as I saw more items (tense, switched the order of a sentence around, etc). I think it's good to go!

StraubCreative commented on content/applications/finance/expenses.rst:
```diff
@@ -153,14 +161,37 @@ into an :guilabel:`Edit` button).
+Rather than manually inputting all of the information for an expense, expenses can be created
+automatically by scanning a PDF receipt.
+
+First, in the main *Expenses* app dashboard view (this view can also be accessed from
```
Yea menuselection could work or we could mimic what we did for 16 and do a gui here.

StraubCreative reviewed (changes requested):
Hi @larm-odoo looks good, just one quick tag to look at.

As well, before we merge can you delete unused images in the file's media folder, please?
Example: `create-new-expense.png`

I think after that we can merge. Tag me again when we're ready :)

StraubCreative reviewed (approved):
@fw-bot ignore
@robodoo r+

robodoo commented:
@larm-odoo @StraubCreative staging failed: ci/runbot on 391003569013df53bd10ef9757f4ab392edf6057 (view more at https://runbot.odoo.com/runbot/build/46867680)

StraubCreative commented:
@robodoo retry

robodoo commented:
@larm-odoo @StraubCreative staging failed: ci/runbot on 0d2457a24ac0880de739f28b9a02eac7aaa0e547 (view more at https://runbot.odoo.com/runbot/build/46868684)

StraubCreative commented:
@robodoo retry

robodoo commented:
@larm-odoo @StraubCreative staging failed: ci/runbot on 21e7e247217a34debe56cbd092e2ee93183e0b95 (view more at https://runbot.odoo.com/runbot/build/46890662)

d-fence commented:
@robodoo retry
The non-related failing test in design-theme is now disabled.
